File: Servidor/AtHomeTFGCarlosHernandez/src/main/java/org/tfg/athometfgcarloshernandez/spring/model/response/GetCasasResponseDTO.java
package org.tfg.athometfgcarloshernandez.spring.model.response;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.tfg.athometfgcarloshernandez.spring.model.CasaDetallesDTO;

import java.util.List;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class GetCasasResponseDTO {
    private List<CasaDetallesDTO> casas;
}

File: Cliente/AtHomeTFGAndroidCarlosHernandez/app/src/main/java/com/example/athometfgandroidcarloshernandez/data/model/response/GetCasasResponseDTO.kt
package com.example.athometfgandroidcarloshernandez.data.model.response

import com.example.athometfgandroidcarloshernandez.data.model.CasaDetallesDTO

data class GetCasasResponseDTO (
    val casas: List<CasaDetallesDTO> = emptyList()
)